src/clinical/api/clinical-api.ts

```diff
@@ -186,15 +183,39 @@ class ClinicalController {
 
 		const donorEntityData = await service.getDonorEntityData(donorIds);
 
+		const entityRecords = donorEntityData[0].records;
+
+		// File Table can request multiple programs
+		const donorPrograms = entityRecords
+			.map((record) => record.program_id as string)
+			.filter((programId, index, array) => programId && array.indexOf(programId) === index);
```

Comment:
This filter isn't doing anything.
It's always going to be `true` because you're filtering on the array already and `index` is the current iteration index.
So `arrayIndexOf(programId)` and `index` are the same value.

Reply:
great! If you fancy adding a comment I'd appreciate it, as I didn't grok it